matrix_sdk/src/client.rs

```diff
@@ -459,6 +461,13 @@ impl RequestConfig {
         self.retry_timeout = Some(retry_timeout);
         self
     }
+
+    /// Force sending authorization even if the endpoint does not require it. Default is only
+    /// sending authorization if it is required
+    pub fn force_auth(mut self) -> Self {
```

Comment:
When would a user use this?

Reply:
Not sure actually. Maybe a weird custom homeserver or something? Would also be fine with making it private.

Reply:
Yeah, let's not expose this unless it's absolutely needed.